src/compiler/diagnosticMessages.json

```diff
@@ -4132,6 +4132,10 @@
         "category": "Error",
         "code": 4124
     },
+    "The values of '{0}.{1}' differ in their declarations, where '{2}' was expected but '{3}' was given.": {

```

Comment:
The problem is that each of them have identical names between members and enums 😄

Reply:
Unless I fully qualified the name - but the earlier error message will have done that anyway.

Reply:
I agree with Nathan's suggestion that it would be better to use the fully qualified names when talking about the members and the different values, even though you can figure it out with the current error message + elaboration.

Reply:
It's definitely redundant, but the redundancy makes it easier to read in this case because there's less context to piece together, and less work to do overall to read the message.